### PR TITLE
feat: Add 'mobile: getAppStrings' extension

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1044,20 +1044,34 @@ wifi | booolean | no | Either to enable or disable Wi-Fi. | false
 data | booolean | no | Either to enable or disable mobile data. | false
 airplaneMode | booolean | no | Either to enable or disable Airplane Mode. | false
 
+### mobile: getAppStrings
+
+Retrieves string resources for the given app language. An error is thrown if strings cannot be fetched or no strings exist
+for the given language abbreviation.
```
Comment (KazuCocoa) [inline README.md:1050]: ah, need to address the supported version?

Comment (mykola-mokhnach) [reply README.md:1050]: not sure what you mean?

Comment (KazuCocoa) [reply README.md:1050]: I mean this is available since uia2 driver version 2.15.0

Comment (mykola-mokhnach) [reply README.md]: added

**Pull request merged**